src/server/api/routes/users/v2/wallet.js

```diff
@@ -58,4 +62,82 @@ router.get('/wallet', async (req, res, next) => {
   }
 });
 
+router.post('/wallet/evm/migrate', async (req, res, next) => {
+  try {
+    const key = req.headers['x-likerland-api-key'];
+    const { wallet } = req.query;
+    const { evmWallet } = req.body;
```

Comment:
why not all put in body?

Reply:
Trying to keep consistent that `const { wallet } = req.query;` is the resource being targeted, `evmWallet` is part of the payload for POST